pa11ycrawler/cmdline.py

```diff
@@ -204,9 +205,13 @@ def main():
     parser = cli()
     args = parser.parse_args()
     settings = args_to_settings(args)
+    configure_logging({
+        'LOG_FORMAT': '%(asctime)s [%(levelname)s] [%(name)s]: %(message)s',
```

Comment:
Nope, you're not missing something -- I was.    You can use `settings.get()` for `LOG_LEVEL`.  I've updated.  Thanks!